src/utils/_utils.js

```diff
@@ -0,0 +1,18 @@
+const handlePadding = paddingOption => {
```

Comment:
Made this a handy reusable function as it'll be used in a few other places down the line